core-bundle/contao/drivers/DC_Folder.php

```diff
@@ -419,6 +419,10 @@ public function showAll()
 		{
 			// Show an empty tree if there are no search results
 		}
+		elseif (empty($this->arrFilemounts) && !\is_array($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['root'] ?? null) && ($GLOBALS['TL_DCA'][$this->strTable]['list']['sorting']['root'] ?? null) !== false)
+		{
+			$return .= $this->generateTree($this->strRootDir . '/' . $this->strUploadPath, 0, false, true, $blnClipboard ? $arrClipboard : false, $arrFound);
+		}
```

Comment:
FYI, this is the exact same code as before. It was removed in https://github.com/contao/contao/pull/6124/files#diff-7343995af85db55c2af9e4328df59608eb269260447eea2aa1f1492da27466c2L425-L428 but it should not have.